admin-dev/themes/new-theme/js/pages/product/components/combination-list/bulk-choices-selector.ts

```diff
@@ -59,21 +87,38 @@ export default class BulkChoicesSelector {
    * Delegated event listener on tabContainer, because every checkbox is re-rendered with dynamic pagination
    */
   private listenCheckboxesChange(): void {
+    this.eventEmitter.on(CombinationEvents.listRendered, () => {
```

Comment:
Does it mean when the list is rendered (so on page change, or filter change) You keep the initial selected state of the all cehcboxes?

I'm not sure about this behaviour which may be risky and result in unexpected behaviour, I think the safest when the page is updated is to unselect everything all the time. But I may be ignoring a use case here 🤷‍♂️

Reply:
Also isn't this event watch redundant with one on L83?